config/example/admins.txt

```diff
@@ -5,4 +5,4 @@
 # Ranks can be anything defined in admin_ranks.txt           	     #
 ######################################################################
 
-# example - Hosting Provider
+# example - Hosting Provider
```

Comment:
Do please revert this change. Itll mess up the history etc otherwise.
You can change it using notepad or such and just add a new line if vscode is being a shit.
Or you can setup vscode to add new lines at the end of files